pkg/builder/csolution/builder.go

```diff
@@ -293,6 +294,59 @@ func (b CSolutionBuilder) listToolchains(quiet bool) (toolchains []string, err e
 	return toolchains, nil
 }
 
+func (b CSolutionBuilder) listEnvironment(quiet bool) (envConfigs []string, err error) {
+	// get installer path and version number
+	getInstallerInfo := func(name string) string {
```

Comment:
It seems it is related to "installed executable" instead of "installer", maybe `getInstalledExeInfo` is a better name? Similar to the function `utils.GetInstalledExePath`.

Reply:
Agreed. [Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild/pull/62/commits/183db637e7d6c9e4158bf67cb009bfc4171c9a8f)